Library/Homebrew/cmd/cleanup.rb

```diff
@@ -62,6 +62,10 @@ def cleanup_cache
         cleanup_path(path) { path.unlink }
         next
       end
+      if path.to_s.include?("java_cache")
```

Comment:
+1 for the 100% safe directory name as well. Can we still have some :cake:? :smile: